state-chain/pallets/cf-pools/src/benchmarking.rs

```diff
@@ -47,6 +47,37 @@ fn new_lp_account<T: Chainflip + Config>() -> T::AccountId {
 mod benchmarks {
 	use super::*;
 
+	// Create some orders so the cost of sweeping is taken into account
+	fn create_some_orders<T: Config>(caller: T::AccountId) {
+		T::LpBalance::credit_account(&caller, Asset::Eth, 1_000_000_000);
+		T::LpBalance::credit_account(&caller, Asset::Usdc, 1_000_000_000);
+
+		for i in 1..=3 {
+			assert_ok!(Pallet::<T>::set_range_order(
+				RawOrigin::Signed(caller.clone()).into(),
+				Asset::Eth,
+				Asset::Usdc,
+				i as u64,
+				Some(-i..i),
+				RangeOrderSize::AssetAmounts {
+					maximum: AssetAmounts { base: 1_000_000, quote: 1_000_000 },
+					minimum: AssetAmounts { base: 500_000, quote: 500_000 },
+				},
+			));
+		}
+		for i in 1_u64..=10 {
```

Comment:
Doesn't have to be in this PR, but it would be good to know how this scales: we tripled the weight estimates with this, but how realistic are these numbers, and is there a point at which we need to start worrying about performance... ?